README.md

```diff
@@ -307,21 +320,18 @@ paths:
       hide: true
 ```
 
-#### Example of spec file
+## Example of acceptance spec file
```

Comment:
This line is specifically an example for the openapi stuff - so I'd leave it as is and then make a more generic section.